<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MplMainWindow</class>
 <widget class="QMainWindow" name="MplMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="mpllineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="mplpushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="MplWidget" name="mpl" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>571</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="mplmenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mplactionOpen"/>
    <addaction name="separator"/>
    <addaction name="mplactionQuit"/>
   </widget>
   <addaction name="mplmenuFile"/>
  </widget>
  <action name="mplactionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="mplactionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.py</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
